omnigibson/objects/controllable_object.py

```diff
@@ -576,7 +576,7 @@ def get_control_dict(self):
         # TODO: Move gravity force computation dummy to this class instead of BaseRobot
         fcns["gravity_force"] = lambda: (
             ControllableObjectViewAPI.get_generalized_gravity_forces(self.articulation_root_path)
-            if self.fixed_base
+            if self.fixed_base or self._dummy is None
```

Comment:
why is this line needed?

Reply:
I believe that self._dummy is None should be there to account for the fact that dummy is not loaded at this stage of the simulator initialization. However, dummy will be loaded later, after the gravity force computation. 
I also resolved this TODO, moving the dummy initialization from the BaseRobot to ControllableObject.